packages/sdk/src/modules/moduleFromEndpoints/types.ts

```diff
@@ -134,6 +173,32 @@ export type Options = {
    * Default request config for each request.
    */
   defaultRequestConfig?: IncomingConfig;
+
+  /**
+   * An optional custom error handler for HTTP requests.
+   *
+   * If not provided, errors will be thrown as is.
```

Comment:
For the sake of our API Reference docs, I would keep the main description short (An optional custom error handler for HTTP requests.) and include everything else in @remarks.

```suggestion
   * @remarks
   * If not provided, errors will be thrown as is.
```

Otherwise, we will end up with lengthy texts in the `Description` section

![image](https://github.com/vuestorefront/vue-storefront/assets/39009379/2e2f8dd1-f4f5-47e0-993f-9db6b3ec76e9)

Reply:
Done